contents/customers/4dayweek.md

```diff
@@ -0,0 +1,66 @@
+---
+title: How 4DayWeek increased conversion by 4% with one A/B experiment
+customer: 4DayWeek
+logo: ../images/customers/4dayweek/4dayweek-logo.png
+logoDark: ../images/customers/4dayweek/4dayweek-logo-dark.png
+featuredImage: ../images/customers/4dayweek/4dayweek_featured.png
+industries:
+    - Recruitment
+users:
+    - Marketing
+    - Leadership
+    - Founder
+toolsUsed:
+    - Experiments
+    - Product analytics
+date: 2023-09-15
+---
+
+[4DayWeek](https://4dayweek.io/) is the world’s largest job board for employers offering a better work-life balance than the traditional 9 to 5. Created by solo founder Phil McParlane, the platform has experience incredible growth since it was launched and is regularly featured in publications such as Time, Fortune and Bloomberg — success which Phil puts partly down to his decision to build in public. 
+
+“Building in public is a great way to get validation from the community,” says Phil. “It’s especially important in the early days of launching a product, when you want to get feedback, encourage discussion and ensure you’re staying accountable to all your promises!”
+
+The fact that Phil builds in public also made him naturally curious about PostHog, which also prides itself on its [transparency and open source culture](/handbook/company/values). 
```

Comment:
```suggestion
The fact that Phil builds in public made him naturally curious about PostHog, because we also value [transparency and building in public](/handbook/company/values). 
```

Seem weird to use third-person when referring to ourselves